src/_includes/header.html

```diff
@@ -84,6 +84,7 @@
             <li role="separator" class="divider"></li>
             <li class="dropdown-header">Technical References</li>
             <li><a href="{{ site.baseurl }}/contribute/design-principles/">Design Principles</a></li>
+	    <li><a href="{{ site.baseurl }}/contribute/portability/">Portability Framework</a></li>
```

Comment:
My reading of the Jenkins log is that the job definition is fixed, but that the `site.baseurl` is set to http://beam.apache.org during link validation, rather than the actual base of the test run. So I think to make mergebot happy we can do this in a follow-up PR, or we have to manually merge, ignoring mergebot's complaints.

Reply:
Ok. I'm just doing what the other pages do here.

Reply:
Oh, yes, the change is right. The staged menu looks good.